.github/ISSUE_TEMPLATE/5.priority-support.yml

```
@@ -0,0 +1,39 @@
+name: 'Priority support: SLA ⏰'
+description: I'm a Premium plan user and I have the priority support add-on, I can't find a solution to my problem with MUI X.
+title: '[question] '
+labels: ['status: needs triage', 'support: commercial', 'support: unknown']
```

```suggestion
labels: ['status: needs triage', 'support: unknown']
```

Why do we need this able at all?

---

In X we use it to determine if the customer is a paying one but I guess it's not really needed since if you verify your support key you ARE a paying customer.